lms/djangoapps/entitlements/models.py

```diff
@@ -0,0 +1,69 @@
+from django.contrib.auth.models import User
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+from jsonfield import JSONField
+
+from .registry import registry
+from .scope import ScopeFactory
+
+
+class EntitlementModel(models.Model):
```

Comment:
Maybe I missed something obvious, but why is this named `EntitlementModel` versus simply `Entitlement`?